screens/Scan/SuccessfullySharedModal.tsx

```diff
@@ -41,13 +42,38 @@ export const SharingSuccessModal: React.FC<
             {sendVcScreenController.receiverInfo.name}
           </Text>
         </Column>
-        <Column margin="0 0  0">
-          <Button
-            type="gradient"
-            title={t('ScanScreen:status.accepted.gotohome')}
-            onPress={scanLayoutController.DISMISS}
-          />
-        </Column>
+        <Row
+          align="space-evenly"
+          style={{marginBottom: Dimensions.get('screen').height * 0.06}}>
+          <Pressable>
+            <Icon
+              name="home"
+              color={Theme.Colors.Icon}
+              size={33}
+              containerStyle={
+                Theme.SelectVcOverlayStyles.sharedSuccessfullyIconStyle
+              }
+              onPress={scanLayoutController.DISMISS}
+            />
+            <Text align="center" weight="bold">
+              {t('ScanScreen:status.accepted.home')}
+            </Text>
+          </Pressable>
+          <Pressable>
+            <Icon
+              name="history"
+              color={Theme.Colors.Icon}
+              size={33}
+              containerStyle={
+                Theme.SelectVcOverlayStyles.sharedSuccessfullyIconStyle
+              }
+              onPress={scanLayoutController.GOTO_HISTORY}
+            />
+            <Text align="center" weight="bold">
+              {t('ScanScreen:status.accepted.history')}
```

Comment:
Can we use useTranslation function (in LINE 17) here rather than sending key ScanScreen?
`t('status.accepted.home')`

Also can we change if any similar occurrences are available in this file?

Reply:
It's better to add everywhere as per the respective flow in all app screens. For any future corrections or re-change of words.